spring-grpc-spring-boot-autoconfigure/src/main/java/org/springframework/grpc/autoconfigure/server/GrpcServerProperties.java

```diff
@@ -24,7 +24,25 @@
 @ConfigurationProperties(prefix = "spring.grpc.server")
 public class GrpcServerProperties {
 
-	private String address = "*";
+	/**
+	 * Server should listen to any IPv4 and IPv6 address.
+	 */
+	public static final String ANY_IP_ADDRESS = "*";
+
+	/**
+	 * Server should listen to any IPv4 address.
+	 */
+	public static final String ANY_IPv4_ADDRESS = "0.0.0.0";
```

Comment:
**[UNRELATED]** We are not using these last 2 yet. It helped me to see the constants though, named accordingly. I am happy to remove them until we actually use them though.